{{cookiecutter.project_name}}/docs/conf.py

```diff
@@ -16,8 +16,9 @@
 
 # -- Project information -----------------------------------------------------
 
-info = metadata("{{cookiecutter.project_name}}")
-project = info["Name"]
+project_name = "{{cookiecutter.project_name}}"
+info = metadata(project_name)
+package_name = "{{ cookiecutter.package_name }}"
```

Comment:
```suggestion
project_name = info["Name"]
github_repo = info["urls"]["Source"]
```

package_name is never used as far as I can tell and why not get the project name from the `pyproject.toml` to have a single point of truth?